clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1194,6 +1196,13 @@ linkDeviceLibFiles(SmallVectorImpl<StringRef> &InputFiles,
   return *OutFileOrErr;
 }
 
+/// This function is used to link all SYCL device input files into a single
```

Comment:
Could we specify all of this new documentation with doxygen, like the rest of LLVM/clang? Then we should be able to get properly rendered documentation.

Reply:
I am adding detailed code comments. Again, this is another step that we intend to perform as part of pre-upstreaming effort. Thanks

Reply:
Okay, no problem.